### PR TITLE
[W6.4h][W10-B4]Joshua Lee Kai Sheng 

### PR DESCRIPTION
**Pull request by jlks96**
LO-Interfaces -- Implemented Printable interface and added getPrintableString method in ReadOnlyPerson interface

### CODE REVIEW

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -55,4 +55,13 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    @Override
+    public String getPrintableString() {
+        if (!isPrivate()) {
```
Comment (tshradheya) [inline src/seedu/addressbook/data/person/Address.java:61]: Good thinking by taking care of this condition

File: src/seedu/addressbook/data/person/Address.java [hunk 2]
```diff
@@ -55,4 +55,13 @@ public int hashCode() {
...
+            return " Address: " + this.value;
+        } else {
+            return "";
```
Comment (tshradheya) [inline src/seedu/addressbook/data/person/Address.java:64]: Maybe return `Private Data` instead of empty string?

**Review by tshradheya — COMMENTED** [review]
Perfect 🎉

**Comment by jlks96** [comment]
Thanks. Acknowledged and closed.

**Pull request closed without merging**